dbcon/mysql/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ include_directories( ${ENGINE_COMMON_INCLUDES}
                      /usr/include/libxml2
                      ${SERVER_SOURCE_ROOT_DIR}/storage/maria )
 
+SET(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/../../../)
```

Comment:
And where ha_columnstore.so now goes?

Reply:
To the place when MTR expects to find it.
```
ls storage/*/*.so|cat
storage/archive/ha_archive.so
storage/blackhole/ha_blackhole.so
storage/columnstore/ha_columnstore.so   <--- HERE IT IS
storage/connect/ha_connect.so
storage/example/ha_example.so
storage/federated/ha_federated.so
storage/federatedx/ha_federatedx.so
storage/maria/ha_s3.so
storage/mroonga/ha_mroonga.so
storage/rocksdb/ha_rocksdb.so
storage/sphinx/ha_sphinx.so
storage/spider/ha_spider.so
storage/test_sql_discovery/ha_test_sql_discovery.so

```